### PR TITLE
[FarmBeats, Test] Hardcoded endpoint for statically set up resource

### PR DESCRIPTION
**Pull request by agniveshadhikari**
Set up [resource](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/5bae917c-5fc9-40c5-923d-adbd52759048/resourceGroups/static-fb-instances/providers/Microsoft.AgFoodPlatform/farmBeats/internal-sdk-live-test-py/resourceOverviewId) and assigned the right role to the test principal.

Removed the deployment of farmbeats instance and the role assignment from the template, and hardcoded the endpoint for the resource above into the outputs from the template.

### CODE REVIEW

**Comment by agniveshadhikari** [comment]
@benbp cc

**Review by benbp — APPROVED** [review]

File: sdk/agrifood/test-resources.json [hunk 1]
```diff
@@ -41,30 +41,11 @@
...
-        "principalId": "[parameters('testApplicationOid')]",
-        "scope": "[resourceGroup().id]"
-      }
-    }
   ],
   "outputs": {
     "AGRIFOOD_ENDPOINT": {
       "type": "string",
-      "value": "[reference(resourceId('Microsoft.AgFoodPlatform/farmbeats', variables('resourceName'))).instanceUri]"
+      "value": "https://internal-sdk-live-test-py.farmbeats.azure.net"
```
Comment (benbp) [inline sdk/agrifood/test-resources.json:48]: Does the test identity have access to this resource?

Comment (agniveshadhikari) [reply sdk/agrifood/test-resources.json:48]: Yep, gave it access.

**Comment by agniveshadhikari** [comment]
@lmazuel We are also ready to merge this

**Review by iscai-msft — APPROVED** [review]

**Pull request merged**